cmd/api/src/model/appcfg/parameter.go

```diff
@@ -89,94 +139,69 @@ type ParameterService interface {
-				Description: CitrixRDPSupportDescription,
-				Value:       citrixRDPSupportValue,
-			},
-		}, nil
-	}
-}
+// PasswordExpirationWindow
 
 type PasswordExpiration struct {
-	Duration string `json:"duration"`
+	Duration time.Duration `json:"duration"`
 }
 
-func (s PasswordExpiration) ParseDuration() (time.Duration, error) {
-	if duration, err := iso8601.FromString(s.Duration); err != nil {
-		return 0, err
+// Because PasswordExpiration are stored as ISO strings, but we want to use them as durations, we override UnmarshalJSON to handle the conversion
+func (s *PasswordExpiration) UnmarshalJSON(data []byte) error {
+	pDb := struct {
+		Duration string `json:"duration,omitempty"`
+	}{}
+
+	if err := json.Unmarshal(data, &pDb); err != nil {
+		return fmt.Errorf("error unmarshaling data for PasswordExpiration: %w", err)
 	} else {
-		return duration.ToDuration(), nil
+		if duration, err := iso8601.FromString(pDb.Duration); err != nil {
+			return err
+		} else {
+			s.Duration = duration.ToDuration()
+		}
+
+		return nil
 	}
 }
 
-func GetPasswordExpiration(ctx context.Context, service ParameterService) (time.Duration, error) {
+func GetPasswordExpiration(ctx context.Context, service ParameterService) time.Duration {
 	var expiration PasswordExpiration
 
 	if cfg, err := service.GetConfigurationParameter(ctx, PasswordExpirationWindow); err != nil {
-		return 0, err
+		log.Warnf("Failed to fetch password expiratio configuration; returning default values")
+		return DefaultPasswordExpirationWindow
 	} else if err := cfg.Map(&expiration); err != nil {
-		return 0, err
-	} else {
-		return expiration.ParseDuration()
+		log.Warnf("Invalid password expiration configuration supplied; returning default values")
+		return DefaultPasswordExpirationWindow
 	}
+
+	return expiration.Duration
 }
 
+// Neo4jConfigs
+
 type Neo4jParameters struct {
 	WriteFlushSize int `json:"write_flush_size,omitempty"`
 	BatchWriteSize int `json:"batch_write_size,omitempty"`
 }
 
 func GetNeo4jParameters(ctx context.Context, service ParameterService) Neo4jParameters {
-	var result Neo4jParameters
+	var result = Neo4jParameters{
+		WriteFlushSize: neo4j.DefaultWriteFlushSize,
+		BatchWriteSize: neo4j.DefaultBatchWriteSize,
+	}
 
 	if neo4jParametersCfg, err := service.GetConfigurationParameter(ctx, Neo4jConfigs); err != nil {
-		log.Errorf("Failed to fetch neo4j configuration; returning default values")
-		result = Neo4jParameters{
-			WriteFlushSize: neo4j.DefaultWriteFlushSize,
-			BatchWriteSize: neo4j.DefaultBatchWriteSize,
-		}
-	} else if err = neo4jParametersCfg.Map(result); err != nil {
-		log.Errorf("Invalid neo4j configuration supplied; returning default values")
-		result = Neo4jParameters{
-			WriteFlushSize: neo4j.DefaultWriteFlushSize,
-			BatchWriteSize: neo4j.DefaultBatchWriteSize,
-		}
+		log.Warnf("Failed to fetch neo4j configuration; returning default values")
+	} else if err = neo4jParametersCfg.Map(&result); err != nil {
```

Comment:
This fixes the loading of neo4j parameters that's been broken since the dawn of time 🌅